### DISCUSSION TITLE
False positive warning: "No balance change found. This transaction maybe malicious"

### DISCUSSION THREAD

**Discussion by romanix** [phantom/docs#120] [General]
Hi all,

Our app gets a warning in Phantom Mobile: "No balance change found. This transaction maybe malicious". See attached screenshot. What can we do to prevent this warning?

The transaction is not malicious but it is a transaction that is transferring a custom SPL token. 

[Transaction on Solana.fm](https://solana.fm/tx/3EbkjX93uRbMZNwh18sKWMWx4WZn2zq8xuiTNNoAht8GFHbr2eyZT9CHM6rK6DPgYhRsVx8cmjFuS6DANGPNhLpm?cluster=mainnet-solanafmbeta)

App URL: [https://app.gbits.io/](https://app.gbits.io/)

Warning: see screenshot of Phantom Mobile
![phantom-warning-2023-11-12](https://github.com/phantom/docs/assets/110880/0ebf9032-bcd3-428c-b7c7-c7ffefe7874f)

**Comment by adamdelphantom** [comment]
Hi @romanix,

I'm looking into this. I'll follow up with you once I have an update.

**Reply by adamdelphantom** [reply]
@romanix Can you provide the associated program addresses, please? We're having trouble looking into this since it requires a Swiss QR invoice.

**Comment by romanix** [comment]
Hi Adam,
Thank your for the response. I've tried multiple times to reproduce that warning and meanwhile doesn't show up anymore. Therefore I closed this issue.

**Reply by romanix** [reply]
![swiss-qr-invoice-USDC](https://github.com/phantom/docs/assets/110880/73d46824-d5fe-4ae9-8fa2-92ef4ee8f6ce)
